[IMP] Allows sorting filter menu in readonly mode (#8196)
odoo/o-spreadsheet: doc/extending/command.md

### Read-Only Mode

> All commands are allowed in readonly mode. Since the spreadsheet state cannot be modified , we use a specific transport service (ReadonlyTransportFilter) blocking revision to be send to the server.

Reviewer: ```suggestion
All commands are allowed in readonly mode. Because the spreadsheet’s state cannot be changed, a specific transport service (ReadonlyTransportFilter) is used to prevent revisions from being sent to the server.
```
